docs/buildingpex.rst

```diff
@@ -100,7 +100,7 @@ way to freeze a virtualenv into a PEX file:
 
 .. code-block:: console
 
-    $ pex -r <(pip freeze) -o my_application.pex
+    $ pex $(pip freeze) -o my_application.pex
```

Comment:
You'll need to update the lead in paragraph above too: https://github.com/pantsbuild/pex/pull/1323/files#diff-ade16e5bd61e6943865733995fbf8a75c6cf7b5e270b583bc3f51606aec4011bR98-R99

As it stands now its confusing, since the paragraph is about `-r` and this freeze reccomendation has nothing to do with that any more.

Reply:
Good catch, thanks!

As you can see, I only read the example 😄